.github/workflows/optimize-labeled.yml

```diff
@@ -0,0 +1,40 @@
+name: Assign labeled issues to Optimize project
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-to-projects:
+    name: Add issue to Optimize project if relevant component label added
+    runs-on: ubuntu-latest
+    if: ${{ github.event.label.name == 'component/optimize' }}
+    steps:
+      - id: get_project_count
+        uses: octokit/graphql-action@v2.3.2
+        with:
+          # API https://docs.github.com/en/graphql/reference/objects#issue
+          query: |
+            query getProjectCount($owner:String!, $repo:String!, $issue: Int!) {
+              repository(name: $repo, owner: $owner) {
+                issue: issue(number: $issue) {
+                  projectsV2 {
+                    totalCount
+                  }
+                }
+              }
+            }
+          variables: |
+            owner: "camunda"
+            repo: "zeebe"
```

Comment:
This is now `camunda´ (technically works with `zeebe` but better to update)

Reply:
renamed